### ISSUE TITLE
Server-side decryption

### ISSUE THREAD

**Issue by kkwaiser** [travist/jsencrypt#101]
Tom Wu's referenced example gives an openssl command to use for decrypting the public-key encrypted string. Can you clarify if it is possible (or what the steps are) to decrypt strings encrypted with this library?

To be clear, an value entered on the client is encrypted with a public key. I would like to decrypt this Base64 string using the private key using openssl.

**Comment by kkwaiser** [comment]
I was able to decrypt JSEncrypted values using OpenSSL using the following commands:

> openssl enc -a -d -in "encrypted.txt" -out "encrypted.txt.bin"
> openssl rsautl -decrypt -in encrypted.txt.bin -inkey key.pem -out decrypt.txt
 
Where encrypted.txt holds the encrypted version of some text string and key.pem holds the private key.

A lot of my difficulty was caused by not knowing the following:
encrypted.txt needs line breaks every 64 characters, no header, etc
key.pem - has -----BEGIN RSA PRIVATE KEY----- and -----END RSA PRIVATE KEY----- header and footer on lines separate from key values. Key needs line breaks every 64 characters.
